workspace/utilities/lib/share-link.xsl

```diff
@@ -93,4 +93,31 @@
 			<xsl:with-param name="content" select="$content"/>
 		</xsl:call-template>
 	</xsl:template>
+	
+	<xsl:template name="share-link-linkedIn">
```

Comment:
All of our templates are kebab-case. Why is this one not `share-link-linkedin` or `share-link-linked-in` ?